pkg/test/framework/components/ingress/kube.go

```diff
@@ -119,7 +119,7 @@ func (c *kubeComponent) Address() string {
 	return c.address
 }
 
-func (c *kubeComponent) Call(path string) (CallResponse, error) {
+func (c *kubeComponent) CallWithHeaders(path string, headers map[string]string) (CallResponse, error) {
```

Comment:
Please, either do not change the name of the method, or add CallWithHeaders as an overload.